check/directory.go

```diff
@@ -9,28 +9,50 @@ import (
 )
 
 const (
+	CdktfIndexDirectory = `cdktf`
+
 	DocumentationGlobPattern = `{docs/index.md,docs/{data-sources,guides,resources},website/docs}/**/*`
```

Comment:
I believe this needs to be updated as well to ensure the checks actually run against `docs/cdktf`:

```suggestion
	DocumentationGlobPattern = `{docs/index.md,docs/{cdktf,data-sources,guides,resources},website/docs}/**/*`
```

Reply:
Yeah good point, I slightly tweaked it so that previously ignored files stay ignored.